vividus/src/main/java/org/vividus/steps/SpringStepFactoryBeanPostProcessor.java

```diff
@@ -35,7 +39,15 @@ public Object postProcessAfterInitialization(Object bean, String beanName)
         if (bean instanceof SpringStepFactory)
         {
             List<String> stepBeanNames = BeanFactoryUtils.mergeLists(applicationContext, "stepBeanNames-");
-
+            String duplicateStepBeanNames = stepBeanNames.stream()
+                    .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()))
+                    .entrySet().stream()
+                    .filter(stepBeanNameCount -> stepBeanNameCount.getValue() > 1)
+                    .map(Map.Entry::getKey)
+                    .collect(Collectors.joining(", "));
+            Validate.validState(duplicateStepBeanNames.isEmpty(),
```

Comment:
on the same way as was suggested by @ikalinin1  to avoid dependecy on 3rd party lib on `CollectionUtils` make sense to   replace `Validate#validState` with `if(condition ) { throw new Exc(Stringformat(bla-bla))}` ?

Reply:
please keep `Validate#validState`